code/datums/uplink_item.dm

```diff
@@ -480,7 +487,7 @@ var/list/uplink_items = list()
 	name = "Stimpack"
 	desc = "Stimpacks, a tool of many great heroes, make you nearly immune to stuns and knockdowns for about 5 minutes after injection."
 	item = /obj/item/weapon/reagent_containers/syringe/stimulants
-	cost = 8
+	cost = 5 //it's much, much worse than an adrenal implant

```

Comment:
Are you sure about that? The stimpack was fixed by me a couple weeks ago, it lasts longer than the adrenal implant, the only difference is that you can't use it when stunned.